Sources/SmithyCodegenCore/AST/Node+AST.swift

```diff
@@ -5,24 +5,11 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-/// Contains the value of a Smithy Node, as used in a JSON AST.
-///
-/// Smithy node data is basically the same as the data that can be stored in JSON.
-/// The root of a Smithy node may be of any type, i.e. unlike JSON, the root element is not limited to object or list.
-///
-/// See the definition of node value in the Smithy spec: https://smithy.io/2.0/spec/model.html#node-values
-enum ASTNode {
-    case object([String: ASTNode])
-    case list([ASTNode])
-    case string(String)
-    case number(Double)
-    case boolean(Bool)
-    case null
-}
+import enum Smithy.Node
```

Comment:
`ASTNode` is deleted & `Smithy.Node` is used instead.

The `Decodable` extension below is applied to `Smithy.Node` instead of to `ASTNode`.